scripts/cibuild

```diff
@@ -23,6 +23,14 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
     if [ "${1:-}" = "--help" ]; then
         usage
     else
+        # Install frontend NPM modules
+        docker-compose \
+            run --rm --no-deps app \
+            yarn install
+
+        # Make sure the database is up for Django tests
+        docker-compose up -d database
```

Comment:
Looks like `pwd-garp` does something similar to what we originally did in this project, albeit directly in `tests` ([source](https://github.com/azavea/pwd-garp/blob/93a6b035a1ba0cab607b902d84dd6cdffe32069f/scripts/test#L26-L34)):

```bash
    # Bring up PostgreSQL and Django in a way that respects
    # configured service health checks.
    docker-compose up -d database django

    # Execute Django tests
    docker-compose \
        run --rm --entrypoint python django \
        manage.py test --noinput
```

I'm not entirely sure when the database service comes up in Temperate. My best guess is that the call to remove the linter output file towards the start of `./scripts/test` brings up the database for the first time through the service-level dependency -- since it doesn't call a Django management command it doesn't care if the database service is accepting connections or not ([source](https://github.com/azavea/temperate/blob/develop/scripts/test#L46-L52)):

```bash
        # Python Lint
        echo "Remove linter output file, if present"
        GIT_COMMIT="${GIT_COMMIT}" docker-compose \
                  -f docker-compose.yml \
                  -f docker-compose.test.yml \
                  run --rm --entrypoint rm \
                  django -f violations.txt
```

One way to test this would be to remove this line from Temperate's `cibuild` and see if the tests continue to pass on CI.

Reply:
I think you're right. I think what you did here is 👌.

Thanks for looking into that further.